Example/Backpack/ViewControllers/MapViewController.swift

```diff
@@ -74,7 +80,7 @@ class MapViewController: UIViewController {
 
 extension MapViewController: MKMapViewDelegate {
     func mapView(_ mapView: MKMapView, viewFor annotation: MKAnnotation) -> MKAnnotationView? {
-        return mapView.dequeueReusableAnnotationView(withIdentifier: "Annotation", for: annotation)
+        return mapView.dequeueReusableAnnotationView(withIdentifier: MapViewController.reuseIdentifier, for: annotation)
```

Comment:
Nit: I don't remember the exact syntax but this can be written in a way that doens't directly reference the classes name. `Self.reuseIdentifier` maybe?